source/common/ssl/context_impl.h

```diff
@@ -163,7 +163,6 @@ class ServerContextImpl : public ContextImpl, public ServerContext {
                            HMAC_CTX* hmac_ctx, int encrypt);
 
   Runtime::Loader& runtime_;
```

Comment:
Note: `runtime_` is only used for `alt_alpn_protocols`, so it should be removed as part of this PR.

Reply:
So clang-tidy tells me :-P  Sadly it's taking longer than expected to clean up all of the tests.